ferrischat_webserver/src/users/bots/get_bots_by_user.rs

```diff
@@ -0,0 +1,51 @@
+use actix_web::{HttpResponse, Responder};
+use ferrischat_common::types::{BotsOwnedByUser, InternalServerErrorJson, User, UserFlags};
+
+/// GET `/api/v0/users/{user_id}/bots`
+/// Get all bots owned by the user
+pub async fn get_bots_by_user(auth: crate::Authorization) -> impl Responder {
+    let bigint_user_id = u128_to_bigdecimal!(auth.0);
+
+    let db = get_db_or_fail!();
+
+    let resp = sqlx::query!("SELECT * FROM bots WHERE owner_id = $1", bigint_user_id)
+        .fetch_all(db)
+        .await;
+    return match resp {
```

Comment:
make this more rusty
```suggestion
    match resp {
```